testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -6222,6 +6222,49 @@ void checkRegexpExtract(SqlOperatorFixture f0, FunctionAlias functionAlias) {
     f.checkNull("log(10, cast(null as real))");
   }
 
+  /** Test case for
+   * <a href="https://issues.apache.org/jira/browse/CALCITE-6224">[CALCITE-6224]
+   * Add LOG2 function (enabled in MYSQL, Spark library)</a>. */
+  @Test void testLog2Func() {
+    final SqlOperatorFixture f0 = fixture();
+    f0.checkFails("^log2(4)^",
+        "No match found for function signature LOG2\\(<NUMERIC>\\)", false);
+    final Consumer<SqlOperatorFixture> consumer = f -> {
+      f.setFor(SqlLibraryOperators.LOG2);
```

Comment:
You can set this outside of the consumer for f0 no?

Reply:
I may not understand what you mean. I wrote this based on how other functions are written.

Reply:
I don't know the reason for doing this either.

Reply:
You can set the operator outside of the consumer. There's no point resetting it each time.

Reply:
I try to change the result to null in sqlfunctions, but a null pointer exception will be reported when comparingResultSet is checked.